### PR TITLE
Fix docker setup

### PR DESCRIPTION
**Pull request by erikpetersen-wf**
## Description of Changes

  - Fix docker setup and other small fixes.

## Proposed Testing Steps (If Applicable)

  - Run through readme, make sure everything works properly - test with Postman.
 - Also run `make run-docker`.

## Relevant Issues (If Applicable)

  -

## Maintainer Notifications

  - @danielharasymiw-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ It is worth mentioning that if the inline function requires a reference as one o
 
 | Field       | Map Key   | Type     | Required? | Description
 | ----------- | --------- | -------- | --------- | ------------
```
Comment (danielharasymiw-wf) [inline README.md]: why remove these links?

File: README.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ It is worth mentioning that if the inline function requires a reference as one o
...
-| Function    | `:fn`     | String   | Yes       | Name of one of the [provided functions](#inline-function-documentation) or [a custom extended function](#extending-the-set-of-functions)
+| Function    | `:fn`     | String   | Yes       | Name of one of the provided functions or a custom extended function
```
Comment (danielharasymiw-wf) [inline README.md]: why remove these links?

Comment (erikpetersen-wf) [reply README.md]: They didn't work.

**Comment by danielharasymiw-wf** [comment]
+10

**Comment by danielharasymiw-wf** [comment]
@Workiva/release-management-p

**Pull request merged**